pdfly/cli.py

```diff
@@ -205,6 +206,30 @@ def compress(
     pdfly.compress.main(pdf, output)
 
 
+@entry_point.command(name="uncompress", help=pdfly.uncompress.__doc__)  # type: ignore[misc]
+def uncompress(
+    pdf: Annotated[
+        Path,
+        typer.Argument(
+            exists=True,
+            file_okay=True,
+            dir_okay=False,
+            writable=False,
+            readable=True,
+            resolve_path=True,
+        ),
+    ],
+    output: Annotated[
+        Path,
+        typer.Argument(
+            exists=False,
```

Comment:
```suggestion
```
It can be really useful to override an existing file, I'm not sure this check is needed.
We alread removed it from other subcommands.